packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -111,9 +118,9 @@ export class Popover implements ComponentInterface {
 
   /**
    * Sets the color scheme of the popup
-   * Valid options include: 'primary', 'transparent'
+   * Valid options include: `light`, `dark` and `primary`.
    */
-  @Prop() colorScheme: 'primary' | 'transparent' =
+  @Prop() colorScheme: 'light' | 'dark' | 'primary' =
```

Comment:
The developer experience would be improved if we define consistent apis for ino-tooltip & ino-popover as well. Currently ino-tooltip provides 'light','primary','secondary' & 'transparent' color schemes. It would be nice if both components make use of the same set of options. This would also simplify maintaining a consistend design & implementation.

Reply:
I added a `TippyThemes` type 👍
Did the same change in PR #957